jbpm-human-task/jbpm-human-task-core/src/test/java/org/jbpm/services/task/identity/DBUserGroupCallbackImplTest.java

```diff
@@ -161,44 +205,51 @@ public void testUserGroups() {
 
     @Test
     public void testUserNotExists() {
-
-        DBUserGroupCallbackImpl callback = new DBUserGroupCallbackImpl(props);
         boolean exists = callback.existsUser("mike");
         assertFalse(exists);
     }
 
     @Test
     public void testGroupNotExists() {
-
-        DBUserGroupCallbackImpl callback = new DBUserGroupCallbackImpl(props);
         boolean exists = callback.existsGroup("HR");
         assertFalse(exists);
     }
 
     @Test
     public void testNoUserGroups() {
-
-        DBUserGroupCallbackImpl callback = new DBUserGroupCallbackImpl(props);
         List<String> groups = callback.getGroupsForUser("mike");
         assertNotNull(groups);
         assertEquals(0, groups.size());
 
     }
 
-    @Test(expected = IllegalArgumentException.class)
+    @Test
     public void testInvalidConfiguration() {
-
-        Properties invalidProps = new Properties();
-        DBUserGroupCallbackImpl callback = new DBUserGroupCallbackImpl(invalidProps);
-        callback.getGroupsForUser("mike");
-        fail("Should fail as it does not have valid configuration");
-
+        assertThatThrownBy(() -> {
+            configureWrongProps();
+        }).isInstanceOf(IllegalArgumentException.class)
```

Comment:
This can be replaced with method reference:
```suggestion
        assertThatThrownBy(this::configureWrongProps).isInstanceOf(IllegalArgumentException.class)
```

Reply:
Done